Casa-IoT-keras/README.md

```diff
@@ -0,0 +1,192 @@
+# CASA example - Keras version
+This classsic example of Human Daily Activity Recognition (HDAR) is well suited both as a lightweight test when learning FEDn and developing on FEDn in psedo-distributed mode. A normal high-end laptop or a workstation should be able to sustain at least 15 clients. The example is also useful for general scalability tests in fully distributed mode. 
+
+[comment]: <> (### Provide local training and test data)
+
+[comment]: <> (To download and prepare the partitioned dataset &#40;creates 10 partitions&#41; in ./data/casa{index}/c{index} etc:)
+
+[comment]: <> (https://archive.org/download/all_20210914/all.csv)
+
+[comment]: <> (, each apartment data are distributed over 11 clients)
+
+[comment]: <> (```)
+
+[comment]: <> (python create_data_partitions.py)
+
+[comment]: <> (```)
+## Provide local training and test data
+For large data transfer reason we uploaded a data folder in this use case to archive.org. To test this use case, you need to download prepared data that composed 27 apartments (casa's). In this use-case, two different ways have been provided to generate the dataset partitions (training, and testing set), that gives the user more flexibility, these ways are listed below.
```

Comment:
Remove information about archive.org. "To prepare the data for this use case, which is composed of 27 apartments..."
What is the difference of the first way of partitioning the data and the second, by default create_data_partitions.py create 10 partitions why is the second option needed?